### ISSUE TITLE
Text-layer spans's contain the correct text of pdf but either svg-layer or canvas-layer are blank

### ISSUE THREAD

**Issue by nouredine** [react-pdf-viewer/react-pdf-viewer#1351]
Hi @phuocng,

I'm displaying pdfs as base64 format, but some of them weren't displayed correctly even though i see texts inside spans as bellow 

![image](https://user-images.githubusercontent.com/6216737/207817034-36cc64cd-c991-44e6-960b-a271ee0b081c.png)
that's happening for either generated canvas or SVGs.

When i inspect the svg-layer I notice some strange encoding as bellow

![image](https://user-images.githubusercontent.com/6216737/207823621-63fb6d03-6d6b-4919-aa65-e55a75a0f753.png)

i'm using the following versions of packages
"@react-pdf-viewer/core": "^2.7.2"
"pdfjs-dist": "2.6.347"

i succeeded to upgrade  react-pdf-viewer to version 3 but updating pdfjs to version 2.16.105 or 3 led to a build issue because i think they're targeting es2020, so i kept only the previous versions of packages

In Summary, text-layer spans's contain the correct text of pdf but not the case for either svg-layer or canvas-layer

Note: using the same base64 format string on an online tool for preview as a pdf, works fine. 
 
I appreciate your help

**Comment by nouredine** [comment]
This is a fonts related issue

**Comment by phuocng** [comment]
Thank for your feedback, @nouredine 👍
